### ISSUE TITLE
Feature Request: Apply custom prompts to highlighted text within PDF viewer 

### ISSUE THREAD

**Issue by Awwwrnie** [logancyang/obsidian-copilot#117]
This plugin is amazing. Appreciate your work on it and excited to see what’s next.

Now that obsidian has compatibility for PDFs, I would love to be able to apply prompts to text within the PDF. It would be great if we could highlight text within the PDF and be able to apply a custom prompt to the selection. 

Thanks.

**Comment by logancyang** [comment]
Thanks for reporting this, and sorry for the slow reply.

Copilot v3 is in code freeze, so we're closing this as not planned. v4 is a ground-up rewrite and is launching soon, with a very different architecture around agent mode.

If this is still an issue for you in v4 once it lands, please open a new issue and we'll take a fresh look. 🙏
